anthropic/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "53fe7c3e-57eb-42ca-8e43-ec92c04b6160",
+  "app_id": "anthropic",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Optimize Anthropic usage: cost estimates, prompt sampling and performance metrics.",
```

Comment:
@dpoint01 Due to lack of copy instructions. I duplicated the same description on Openai let me know if you need this changed

Reply:
@barieom can you provide copy here, the above copy us incorrect and OpenAI specific.